alert-common/src/main/java/com/synopsys/integration/alert/common/email/EmailMessagingService.java

```diff
@@ -197,19 +200,19 @@ public void sendMessages(EmailProperties emailProperties, Session session, List<
         }
     }
 
-    private void sendAuthenticated(EmailProperties emailProperties, Message message, Session session) throws MessagingException {
+    private Transport getTransport(EmailProperties emailProperties, Session session) throws MessagingException {
```

Comment:
In truth, there's little "get"-ing going on here. Maybe getAndConnectToTransport?